README.md

```diff
@@ -6,14 +6,68 @@ End-user apps in WSO2 Identity Server
 | :------------ |:------------- 
 | master      | [![Build Status](https://wso2.org/jenkins/view/Dashboard/job/docs/job/docs-is/badge/icon)](https://wso2.org/jenkins/view/Dashboard/job/docs/job/docs-is/) |
 
-## Prerequisite
+## Prerequisites
 
-To run the project on local, it requires [python](https://www.python.org/downloads/) & [mkdocs](https://www.mkdocs.org/)
+To run the project locally, it requires [python](https://www.python.org/downloads/) & [pip](https://pypi.org/project/pip/).
+
+### Install Python
```

Comment:
shall we make this h2

Reply:
Fixed with `2b173a1`

Reply:
Since `Install Python` comes under `Prerequisites` which is a h2, doesn't it have to be a h3? WDYT?

Reply:
Ah yes